Sources/XMLCoder/Decoder/XMLDecoder.swift

```diff
@@ -252,8 +252,12 @@ open class XMLDecoder {
 
     /// A node's decoding type
     public enum NodeDecoding {
+        /// Decodes a node from attributes of form `nodeName="value"`.
         case attribute
+        /// Decodes a node from elements of form `<nodeName>value</nodeName>`.
         case element
+        /// Decodes a node from either elements of form `<nodeName>value</nodeName>` or attributes
+        //// of form `nodeName="value"`.
```

Comment:
Do we guarantee the order in which the check is made?

Reply:
Great catch, I've clarified it in the comment